<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMaterial</class>
 <widget class="QWidget" name="WidgetMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Material:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="materialComboBox"/>
   </item>
   <item>
    <widget class="QComboBox" name="textureComboBox"/>
   </item>
   <item>
    <widget class="QComboBox" name="shaderComboBox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="colorLabel">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorButton">
       <property name="text">
        <string>Change color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="useColor">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Use</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Specular Strength</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="specularStrength">
       <property name="maximum">
        <double>2.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Specular Exponent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="specularExponent">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
